config.js

```diff
@@ -14,6 +14,16 @@ function getConfig() {
     const releaseNotesPrefix = core.getInput('release-notes-prefix');
     const releaseNotesSuffix = core.getInput('release-notes-suffix');
 
+    let releaseNotesPrefixPattern = undefined;
```

Comment:
`= undefined` is unnecessary/redundant. All variables are undefined when declared without an initializer.